client/src/legacy/LeftAndMain.EditForm.js

```diff
@@ -609,7 +609,7 @@ $.entwine('ss', function($){
   });
 
   /**
-   * Add support for key navigation between tabs on the primary/main tab nav
+   * Add support for key navigation between tabs on the secondary nav
    *
    * Note that prior to this, arrow keys did work on secondary tabsets, presumably
    * set by jquery-ui tabs, and they did auto load the panel, though PageUp and PageDown did not.
```

Comment:
```suggestion
   * set by jquery-ui tabs, and they did auto load the panel, though Home and End did not.
```
Probably worth fixing while we're in here.

Reply:
Updated